system_tests/shovel_system_test.go

```diff
@@ -145,6 +146,46 @@ var _ = Describe("Shovel", func() {
 		Expect(k8sClient.Delete(ctx, shovel)).To(Succeed())
 		assertShovelDeleted(shovel)
 	})
+
+	It("with DeletionPolicy=retain, deletes k8s resource but keeps the shovel in RabbitMQ", func() {
```

Comment:
Same as with Federation system tests. I think it's better to surround this `It` with `When` like `When("deletion policy is retain", func() {` because it reads out more naturally.